descriptors/ModuleDescriptor-template.json

```diff
@@ -69,7 +69,10 @@
           "pathPattern": "/invoice/invoices/{id}",
           "permissionsRequired": ["invoice.invoices.item.delete"],
           "modulePermissions": [
-            "invoice-storage.invoices.item.delete"
+            "invoice-storage.invoices.item.get",
+            "invoice-storage.invoices.item.delete",
+            "acquisitions-units-storage.units.collection.get",
```

Comment:
At the moment, the business module is used to obtain units and memberships https://github.com/folio-org/mod-invoice/blob/0c83549805e120b152e5058f3c9cb44d14d8bd35/src/main/java/org/folio/invoices/utils/ResourcePathResolver.java#L32-L33
Either switch to the storage module or use `acquisitions-units.units.collection.get`, `acquisitions-units.memberships.collection.get` permissions

Reply:
Good catch @pautau-s.  There has been some debate in the community regarding BL modules calling other BL modules... I think in this case we should probably just call the storage module, but practically speaking since the acq-unit stuff lives in mod-orders and invoices already has a dependency on orders (BL) - i.e. for updating order paymentStatus, it's not a huge deal if use the acq-unit BL interface

Reply:
Ok - I will change to `acquisitions-units.units.collection.get` and `acquisitions-units.memberships.collection.get`